packages/block-editor/src/components/block-list/block-contextual-toolbar.js

```diff
@@ -41,22 +46,24 @@ function BlockContextualToolbar( { focusOnMount, ...props } ) {
 	}
 
 	// Shifts the toolbar to make room for the parent block selector.
-	const classes = classnames( 'block-editor-block-contextual-toolbar', {
-		'has-parent': hasParents,
-	} );
+	const classes = classnames(
+		'block-editor-block-contextual-toolbar',
+		className,
+		{
+			'has-parent': hasParents,
+		}
+	);
 
 	return (
-		<div className="block-editor-block-contextual-toolbar-wrapper">
```

Comment:
This class is not used anywhere, so it should be safe to remove this useless div.